docs/markdown/Adding-new-projects-to-wrapdb.md

```diff
@@ -132,6 +141,24 @@ Under macOS the command is the following:
 shasum -a 256 path/to/libfoo-1.0.0.tar.gz
 ```
 
+Next you need to add the entries that define what dependencies the
+current project provides. This is important, as it is what makes
+Meson's automatic dependency resolver work. It is done by adding a
+`provide` entry at the end of the `upstream.wrap` file. Using the Ogg
+library as an example, this is what it would look like:
+
+```ini
+[provide]
+ogg = ogg_dep
+```
+
+The `ogg` part on the left refers to the dependency name, which should
+be the same as its Pkg-Config name. `ogg_dep` on the right refers to
+the variable in the build definitions that provides the dependency.
+Most commonly it holds the result of a `declare_dependency` call. If a
+variable of that name is not defined, Meson will exit with a hard
+error.
+
```

Comment:
Could probably link to https://mesonbuild.com/Wrap-dependency-system-manual.html#provide-section.